_posts/guides/2021-12-21-guide_gamemode-info-bhop.md

```diff
@@ -19,14 +19,32 @@ toc_sticky: true
 
 Bhop, short for "***B***unny ***Hop***", is a very popular gamemode found in many games, such as CS:GO, TF2, and others.
 
+In order to maximize the likelihood of inputting the jump exactly when you hit the ground, players used to bind jump to the scroll wheel, and then roll their mouse wheel very fast whenever they were about to hit the ground. This is sometimes known as *traditional* style bhopping. This method has fallen out of favor though, as unless the player can frame-perfectly input every jump command the best the player can hope for is a 50% chance of not losing speed. It's more common now to rely on *autobhop*, an addon or mod which inputs a jump on the exact frame the player hits the ground every time. This functionality is built into MomentumMod
+
 ## History of Bhop
 
-TODO full history of bhop, mentioning defrag probably as the grandpa
+**TODO:** full history of bhop, mentioning defrag probably as the grandpa. Some paragraphs might have to be relocated here, I'm not totally sure where each explanation should be.
+
+## How Bhop Works (Non-Technical)
```

Comment:
Remove (Non-Technical) here since the next header isn't How Bhop Works (Technical)